src/handlers/touch_handler.rs

```diff
@@ -0,0 +1,218 @@
+use bevy::input::touch::{TouchInput, TouchPhase};
```

Comment:
to be renamed to `src/handlers/touch.rs`

Reply:
Also should we call the folders `handlers` it feels generic, can we call it `input` because these are input devices?